kits/hybrid-rrf-search/prompts/hydrid-rrf-search_instructor-llmnode-687_system_0.md

```
@@ -0,0 +1,2 @@
+You are an expert relevance-scoring AI. You will be provided with a user's search query and a JSON array of retrieved passages. Your job is to evaluate how relevant each passage is to the query on a scale of 0.0 to 1.0.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)